Lib/test/test_xml_etree.py

```diff
@@ -2821,9 +2839,11 @@ def tearDown(self):
         super().tearDown()
 
     def test_find_with_mutating(self):
-        e = ET.Element('foo')
-        e.extend([ET.Element('bar')])
-        e.find(MutatingElementPath(e, 'x'))
+        for cls in [MutationDeleteElementPath, MutationClearElementPath]:
+            with self.subTest(cls):
+                e = ET.Element('foo')
+                e.extend([ET.Element('bar')])
+                e.find(cls(e, 'x'))
```

Comment:
We don't test the result? Same question for findtext/findall tests.

Reply:
Err.. I only wanted to check that we don't crash but we could perhaps check the result. I'm worried however that the Python implementation may output something else (but I didn't check actually)